openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -479,6 +479,13 @@ def get(self, request, usage_key_string, *args, **kwargs):
         except InvalidKeyError:
             raise NotFound("Invalid usage key: '{}'.".format(usage_key_string))
 
+        _, user = setup_masquerade(
```

Comment:
needed to setup masquerade for the sequence metadata endpoint

Reply:
It is common to also save the user result in `request.user`, for code that just directly looks at that user. (I know `setup_masquerade` sets an attribute on `request.user`, but that doesn't help naive code.)

Something like:
`_, request.user = setup_masquerade(`

Reply:
changed